src/components/WarrantyForm/WarrantyForm.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import { 
+  Form, 
+  FormContent, 
+  FormInput, 
+  FormOptions 
+} from '../../components/dist';
```

Comment:
Replace with Blocks components from the NPM package

Reply:
Blocks wasn't ready at the time, you will see this update in updated PR.